tests/src/testcommon.hh

```diff
@@ -317,7 +320,9 @@ template <template <typename, int, int> class resType, typename ResultEvaluator>
   sparseAssembler->bind(feRequirements);
   sparseAssembler->bind(Ikarus::DBCOption::Full);
 
-  auto vtkResultFunction   = Ikarus::makeResultVtkFunction<resType, ResultEvaluator>(sparseAssembler);
+  auto vtkResultFunction =
+      Ikarus::makeResultVtkFunction<resType, ResultEvaluator>(sparseAssembler, std::forward<Args>(args)...);
```

Comment:
If we use the refactor from above you would not have to specify the `ResultEvaluator` template arg but only forward the result function coming from outside.